### DISCUSSION TITLE
Create page snapshot AND append to current daily note

### DISCUSSION THREAD

**Discussion by zanzibar21** [coddingtonbear/obsidian-web#205] [Ideas]
Is there a way to combine the capture page snapshot and append to current daily note options into a single action.   When I use the extension I want to create a snapshot and have that saved in a separate folder but also have a line at the end of my daily note that links that newly created note.   Ideally, I'd like to have this as a single step.  Any suggestions?

**Comment by coddingtonbear** [comment]
I'm afraid there's no such functionality!  As things stand right now, you can pick just a single template at a time, and each template can perform just a single interaction with Obsidian.
